lib/cookieman.js

```diff
@@ -114,10 +114,27 @@ function clearAll (name) {
   return cleared
 }
 
+/**
+ * gets the value of a cookie, or returns null
+ *
+ * @public
+ * @param {String} name The cookie name
+ * @returns {Null} or {Boolean} Returns null if the cookie is not found or the cookie's value
+ */
+function cookieValue (name) {
+  var val = null
+  var match = document.cookie.match(new RegExp(name + '=' + '([^;]+)'))
```

Comment:
may want to also `document.cookie && document.cookie.match`. Not sure what browsers will do when cookies aren't set.